README.md

```diff
@@ -19,6 +19,7 @@
   - [Installation](#installation)
   - [Usage](#usage)
   - [Replicating Data](#replicating-data)
+  - [Initialize MyDuck Server with Custom SQLs](#initialize-myduck-server-with-custom-sqls)
```

Comment:
Let's make it shorter: `Using Custom Initialization Scripts`. `MySQL Server` is unneeded.